### PR TITLE
lesson 1: corrections for the internal run

### PR DESCRIPTION
**Pull request by TomasTomecek**

### CODE REVIEW

File: lesson-1-intro-to-k8s/README.md [hunk 1]
```diff
@@ -50,11 +52,12 @@ Welcome to the first lesson of the Red Hat Czech's OpenShift course for beginner
 
 ## Intro
 
-* The difference between OpenShift and Kubernetes.
-* OpenShift can feel complex and intimidating but don't worry! With some effort it will deliver.
+* The difference between [OpenShift](https://github.com/openshift/origin) and [Kubernetes](https://github.com/kubernetes/kubernetes).
```
Comment (stlaz) [inline lesson-1-intro-to-k8s/README.md]: https://github.com/openshift/origin is a test repo

File: lesson-1-intro-to-k8s/Containerfile [hunk 2]
```diff
@@ -1,5 +1,5 @@
-FROM fedora:35
+FROM fedora:37
```
Comment (stlaz) [inline lesson-1-intro-to-k8s/Containerfile]: latest?

Comment (TomasTomecek) [reply lesson-1-intro-to-k8s/Containerfile]: yes, that will make things easier :)

File: lesson-1-intro-to-k8s/README.md (hunk 1 above)
Comment (TomasTomecek) [reply lesson-1-intro-to-k8s/README.md]: oki, I'll use the root of docs then :) https://docs.openshift.com/

**Pull request merged**